setup.py

```diff
@@ -59,6 +59,7 @@ def run_tests(self):
     install_requires=[
         "wrapt",
         "msgpack-python",
+        "setuptools",  # for pkg_resources.parse_version
```

Comment:
I would prefer to keep small our dependency list, and only if really required. We can probably parse and check the version manually instead of adding `setuptools`

Reply:
found a built-in module that can do this

Reply:
making the support optional per previous comment